Gemfile.lock

```diff
@@ -311,69 +312,73 @@ GEM
       actionpack (>= 5.0)
       railties (>= 5.0)
     rexml (3.2.5)
-    rubocop (1.26.1)
+    rubocop (1.35.1)
+      json (~> 2.3)
       parallel (~> 1.10)
-      parser (>= 3.1.0.0)
+      parser (>= 3.1.2.1)
       rainbow (>= 2.2.2, < 4.0)
       regexp_parser (>= 1.8, < 3.0)
-      rexml
-      rubocop-ast (>= 1.16.0, < 2.0)
+      rexml (>= 3.2.5, < 4.0)
+      rubocop-ast (>= 1.20.1, < 2.0)
       ruby-progressbar (~> 1.7)
       unicode-display_width (>= 1.4.0, < 3.0)
-    rubocop-ast (1.16.0)
+    rubocop-ast (1.21.0)
       parser (>= 3.1.1.0)
-    rubocop-performance (1.13.3)
+    rubocop-performance (1.14.3)
       rubocop (>= 1.7.0, < 2.0)
       rubocop-ast (>= 0.4.0)
     ruby-progressbar (1.11.0)
     rubyzip (2.3.2)
-    selenium-webdriver (4.1.0)
+    selenium-webdriver (4.4.0)
       childprocess (>= 0.5, < 5.0)
       rexml (~> 3.2, >= 3.2.5)
-      rubyzip (>= 1.2.2)
-    sidekiq (6.4.1)
-      connection_pool (>= 2.2.2)
+      rubyzip (>= 1.2.2, < 3.0)
+      websocket (~> 1.0)
+    sidekiq (6.5.6)
+      connection_pool (>= 2.2.5)
       rack (~> 2.0)
-      redis (>= 4.2.0)
+      redis (>= 4.5.0, < 5)
     simple_form (5.1.0)
       actionpack (>= 5.2)
       activemodel (>= 5.2)
-    sshkit (1.21.2)
+    sshkit (1.21.3)
       net-scp (>= 1.1.2)
       net-ssh (>= 2.8.0)
     sshkit-interactive (0.3.0)
       sshkit (~> 1.12)
-    standard (1.9.1)
-      rubocop (= 1.26.1)
-      rubocop-performance (= 1.13.3)
-    stimulus-rails (1.0.4)
+    standard (1.16.1)
+      rubocop (= 1.35.1)
+      rubocop-performance (= 1.14.3)
+    stimulus-rails (1.1.0)
       railties (>= 6.0.0)
-    strscan (3.0.1)
+    strscan (3.0.4)
     thor (1.2.1)
-    timeout (0.2.0)
-    turbo-rails (1.0.1)
+    timeout (0.3.0)
+    turbo-rails (1.1.1)
       actionpack (>= 6.0.0)
+      activejob (>= 6.0.0)
       railties (>= 6.0.0)
-    tzinfo (2.0.4)
+    tzinfo (2.0.5)
       concurrent-ruby (~> 1.0)
-    unicode-display_width (2.1.0)
+    unicode-display_width (2.3.0)
     warden (1.2.9)
       rack (>= 2.0.9)
     web-console (4.2.0)
       actionview (>= 6.0.0)
       activemodel (>= 6.0.0)
       bindex (>= 0.4.0)
       railties (>= 6.0.0)
-    webdrivers (5.0.0)
+    webdrivers (5.1.0)
       nokogiri (~> 1.6)
       rubyzip (>= 1.3.0)
       selenium-webdriver (~> 4.0)
+    websocket (1.2.9)
     websocket-driver (0.7.5)
       websocket-extensions (>= 0.1.0)
     websocket-extensions (0.1.5)
     xpath (3.2.0)
       nokogiri (~> 1.8)
-    zeitwerk (2.5.4)
+    zeitwerk (2.6.0)
```

Comment:
### [zeitwerk](https://github.com/fxn/zeitwerk)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 2.5.4 → 2.6.0

[_[change-log](https://github.com/fxn/zeitwerk/blob/master/CHANGELOG.md), [source-code](https://github.com/fxn/zeitwerk), [gem-diff](https://my.diffend.io/gems/zeitwerk/2.5.4/2.6.0)_]





<details>
<summary>Commits</summary>

A change of **43** commits. See the full changes on [the compare page](https://github.com/fxn/zeitwerk/compare/v2.5.4...v2.6.0).

These are the first 10 commits:
- (b602f66) [Use the cget helper here](https://github.com/fxn/zeitwerk/commit/b602f668e7f02a2c92b4fa2f9cc93c3ca8c61e09)
- (95606d1) ["Outstanding Performance Award" Fukuoka Ruby Award 2022](https://github.com/fxn/zeitwerk/commit/95606d14b1d4ce0e74a0fcee9052e0e07d863598)
- (11600bc) [Fixes type signature annotation](https://github.com/fxn/zeitwerk/commit/11600bc7ff494d70b2b3cca72b0ae34f6269dd46)
- (a4d99c0) [Enable warnings when running tests](https://github.com/fxn/zeitwerk/commit/a4d99c0ed8c991355dbe6f6c9af5662cc5591bc1)
- (53fbbb3) [Merge pull request #211 from teoljungberg/enable-warnings](https://github.com/fxn/zeitwerk/commit/53fbbb3aeda54ec780b31c8edff609e65250783b)
- (4097d10) [Let the test suite abort on warnigns](https://github.com/fxn/zeitwerk/commit/4097d101cf527735e35d4295fdb29af09a6cf646)
- (b286030) [Simplify abort on warn](https://github.com/fxn/zeitwerk/commit/b28603042952187c1d5111a5399d0c7eb5f3cf99)
- (8969957) [Decorate Warning.warn as documented](https://github.com/fxn/zeitwerk/commit/89699577ad1f492b1338065f5231e63fcafe7f5e)
- (ac24066) [Let NoWarningsPolicy work with old signatures](https://github.com/fxn/zeitwerk/commit/ac24066eb3480b2cfecf43f4e8ea24ee88fb711b)
- (85a4342) [Use the modern ... as catchall signature](https://github.com/fxn/zeitwerk/commit/85a43423811633ff28b9a5a0651c071d1e5bbfa8)


</details>